tests/Gedmo/Sluggable/TransliterationTest.php

```diff
@@ -45,7 +45,7 @@ public function testInsertedNewSlug(): void
         static::assertSame('tova-e-testovo-zaglavie-bg', $bulgarian->getSlug());
 
         $russian = $repo->findOneBy(['code' => 'ru']);
-        static::assertSame('eto-testovyi-zagolovok-ru', $russian->getSlug());
+        static::assertSame('eto-testovyj-zagolovok-ru', $russian->getSlug());
```

Comment:
This is just transliterated differently between packages.  No idea which is more accurate, but given this is the only test case I actually did have to change, figured I'd point it out.